Simplified/Views/NYPLRoundedButton.swift

```diff
@@ -139,16 +139,26 @@ private let NYPLRoundedButtonPadding: CGFloat = 6.0
     updateColors()
   }
   
+  private var customAccessibilityLabel: String?
+  
   override var accessibilityLabel: String? {
```

Comment:
since the getter is kind of hacky, can you add some docs explaining that the value that we set at line 153 is used only the first time, and after that the accessibility label will be the button label?